lara-typescript/src/interactive-api-client/types.ts

```diff
@@ -404,3 +404,26 @@ export interface ILinkedInteractiveStateResponse<LinkedInteractiveState> {
   listenerId: string;
   interactiveState: LinkedInteractiveState | undefined;
 }
+
+/**
+ * Interface that can be used by interactives to export and consume datasets. For example:
+ * - Vortex interactive is exporting its dataset in the interactive state
+ * - Graph interactive (part of the question-interactives) can observe Vortex interactive state
+ *   (via linked interactive state observing API) and render dataset columns as bar graphs.
+ */
+export interface IDataset {
+  type: "dataset";
+  version: 1;
+  properties: string[];
+  xAxisProp: string;
+  rows: Array<Array<(number | string | undefined | null)>>;
```

Comment:
When the rows are `JSON.stringify`d the undefineds will turn into nulls. That could be confusing for someone expecting undefined to be preserved.

Reply:
Good point, I've fixed that in the last commit.